manual/guides/pyne.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: manual
+title: PyNE
+subtitle: "An important tool in our toolkit"
+permalink: /manual/guides/pyne
+---
+
+# <a name="a brief overview"></a>A Brief Overview
+
+Taken from the [PyNE website](http://pyne.io/index.html), PyNE is intended to be a library used for nuclear engineering. It can be used to help its users "be more productive, have the *best* solvers, have a beautiful API, write really great code, and teach the next generation". The official website provides all the information that one would need, including information about [installation](http://pyne.io/install/index.html) and a [tutorial](http://pyne.io/tutorial/index.html) on using PyNE.
```

Comment:
1. I'm not sure where the quote is from, I couldn't find it on the website. Is this a quote from @katyhuff ?
2. I'd recommend reviewing Katy's writing checklist, particularly about using active voice vs passive voice and "throat clearing."

Reply:
I'll check out the writing and try to make it more active and reduce the filler words. But that quote was from the tutorial page: http://pyne.io/tutorial/00.html

Reply:
@samgdotson let me know what you think. I worked on it a bit more